include/openssl/digest.h

```diff
@@ -346,9 +350,11 @@ struct env_md_ctx_st {
   // manipulate |pctx|.
   const struct evp_md_pctx_ops *pctx_ops;
 
-  // flags, only used to set flag |EVP_MD_CTX_FLAG_KEEP_PKEY_CTX|, so as to let
-  // |*pctx| refrain from being freed when |*pctx| was set externally with
-  // |EVP_MD_CTX_set_pkey_ctx|.
+  // flags is only used for two cases.
+  // 1. Set flag |EVP_MD_CTX_FLAG_KEEP_PKEY_CTX|, so as to let |*pctx| refrain
+  //    from being freed when |*pctx| was set externally with
+  //    |EVP_MD_CTX_set_pkey_ctx|.
+  // 2. Set flag |EVP_MD_CTX_FLAG_NO_INIT| for |EVP_PKEY_HMAC|.
```

Comment:
The comment mentions constant EVP_MD_CTX_FLAG_NO_INIT, but the code checks for EVP_MD_CTX_HMAC. Could it be that EVP_MD_CTX_FLAG_NO_INIT is deprecated? Maybe the comment needs to be updated?

Reply:
Yes I'll change this rn